tests/pywrap-cbs1/input.dat

```diff
@@ -16,7 +16,7 @@ e_cbs = energy(cbs,
             scf_scheme=scf_xtpl_helgaker_3)
 
 compare_values(-7.4326961561955551, e_cbs, 7, "[1] Li ROHF extrapolated energy")  #TEST
-compare_values(3.0, variable('cbs number'), 6, '[1b]')  #TEST
```

Comment:
What motivates the changes here?

Reply:
The "[cbs|nbody|findif] numbers" are ints like "iterations", so it made sense to drop the atol argument. Of course, they're also floats since stored in `P::e.globals`, so I could cast to int first. Overall, this seemed clearer.